beam/tests/test_handling_unit.py

```diff
@@ -19,6 +20,7 @@ def submit_all_purchase_receipts():
 		pr.submit()
 
 
+@pytest.mark.order(1)

```

Comment:
Let's increment these to start at 20 in case there are tests we want to run before these. We also want to add pytest-orer to the other two test files, skip at least 10 in between.